src/pages/2024/barcelona/travel.astro

```diff
@@ -1,67 +1,242 @@
 import { SponsoredBy } from "@components/Marquee";
 import Accommodation from "@modules/Accommodation";
 import MapEmbed from "@components/MapEmbed";
 import Box from "@components/Box";
+import Accordion from "@components/Accordion";
+import imgBarcelona from "@photos/barcelona.jpg";
+import { Image } from "astro:assets";
 import SEO from "@components/SEO";
-import { Code } from "astro:components";
 ---
 
 <Layout namespace="2024/barcelona">
   <SEO slot="head" title="Travel" img={2} bcn />
-  <section class="pb-6 pt-16 md:py-20">
-    <div class="container island text-center">
-      <h1 class="h00 mb-8">Join us in Barcelona</h1>
-      <div class="blockquote">
-        Barcelona is a popular destination, especially during conference season.
-        To ensure your accommodation for the Nextflow Summit, we recommend
-        booking your hotel as soon as possible. We've secured special rates at
-        select hotels. While rooms are available, block bookings are not. Make
-        sure to reserve yours while supplies last using the promo code:
-        <code class="bg-nextflow-800 bg-opacity-30 px-1">nextflow24</code>.
+  <section class="container py-20">
+    <h2 class="h00 mb-12 text-center">Join us in Barcelona</h2>
+    <div class="container mt-5">
+      <div class="flex flex-wrap -m-2 sm:-m-4">
+        <div class="w-full md:w-1/2 p-2 sm:p-4">
+          <div class="rounded-lg overflow-hidden max-h-[300px]">
+            <Image
+              src={imgBarcelona}
+              alt="Nextflow SUMMIT Barcelona"
+              class="object-cover h-[300px] sm:h-full rounded-lg"
+            />
+          </div>
+        </div>
+        <div class="w-full md:w-1/2 p-2 sm:p-4">
+          <Box className="h-[400px] !border-nextflow-600/40">
+            Barcelona, the cosmopolitan capital of Spain’s Catalonia region, is
+            known for its art and architecture. The fantastical Sagrada Família
+            church and other modernist landmarks designed by Antoni Gaudí dot
+            the city. Museu Picasso and Fundació Joan Miró feature modern art by
+            their namesakes. The history museum MUHBA includes several Roman
+            archaeological sites, and the city was, of course, the birthplace of
+            <code class="bg-nextflow-800 bg-opacity-30 px-1">Nextflow</code>.
+          </Box>
+        </div>
       </div>
     </div>
-    <div class="container flex flex-wrap pt-10 md:pt-20">
-      <div class="w-full md:w-1/2 p-2 min-h-[300px] relative">
-        <div class="h-full w-full relative rounded-lg overflow-hidden">
-          <MapEmbed
-            src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d29486.87678206097!2d2.157406824584928!3d41.377369138813016!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x12a4a2523cd5f815%3A0xe4451cff2eaf231d!2sWorld%20Trade%20Center%20Barcelona!5e0!3m2!1sen!2sse!4v1715698687163!5m2!1sen!2sse"
-          />
+
+    <div class="container smaller mt-5">
```

Comment:
I think remove the `smaller` to make this the same width as the rest of the page, right?

```suggestion
    <div class="container mt-5">
```